### PR TITLE
YOLOv5 Sparse Transfer Learning Tutorial and Recipes

### PR DESCRIPTION
**Pull request by markurtz**

### CODE REVIEW

File: integrations/ultralytics-yolov5/recipes/yolov5.transfer_learn_pruned_quantized.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+    start_epoch: 0.0
+    params: __ALL_PRUNABLE__
+    
+quantization_modifiers:
+  - !QuantizationModifier
+    start_epoch: *quantization_start_epoch
+    submodules: [ 'model.0', 'model.1', 'model.2', 'model.3', 'model.4', 'model.5', 'model.6', 'model.7', 'model.8', 'model.9', 'model.10', 'model.11', 'model.12', 'model.13', 'model.14', 'model.15', 'model.16', 'model.17', 'model.18', 'model.19', 'model.20', 'model.21', 'model.22', 'model.23' ]
+---
+
+# YOLOv5 Pruned Quantized Transfer Learning
```
Comment (jeanniefinks) [inline integrations/ultralytics-yolov5/recipes/yolov5.transfer_learn_pruned_quantized.md:74]:
```suggestion
# YOLOv5 Pruned-Quantized Transfer Learning
```

File: integrations/ultralytics-yolov5/tutorials/yolov5_sparse_transfer_learning.md [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+The command uses the `--recipe` argument to encode the proper hyperparams such that SparseML will enforce the sparsity for all layers.
+Without the proper recipes, the zeroed weights will slowly become dense as the model is trained further.
+The recipes are specific to the sparsification type, so the training command will differ based on if you are transfer learning the pruned quantized, pruned, or baseline model.
+
+1) Select the proper command to run based on the model and the sparsification type of the model you chose earlier.
+   - YOLOv5s Pruned transfer learning:
+     ```bash
+     python train.py --data voc.yaml --cfg ../models/yolov5s.yaml --weights zoo:cv/detection/yolov5-s/pytorch/ultralytics/coco/pruned-aggressive_96 --hyp data/hyp.finetune.yaml --recipe ../recipes/yolov5s.transfer_learn_pruned.md
+     ```
+   - YOLOv5s Pruned Quantized transfer learning:
```
Comment (jeanniefinks) [inline integrations/ultralytics-yolov5/tutorials/yolov5_sparse_transfer_learning.md:121]:
```suggestion
   - YOLOv5s Pruned-Quantized transfer learning:
```

File: integrations/ultralytics-yolov5/tutorials/yolov5_sparse_transfer_learning.md [hunk 3]
```diff
@@ -0,0 +1,226 @@
...
+     ```
+   - YOLOv5s Baseline transfer learning:
+     ```bash
+     python train.py --data voc.yaml --cfg ../models/yolov5s.yaml --weights zoo:cv/detection/yolov5-s/pytorch/ultralytics/coco/base-none --hyp data/hyp.finetune.yaml --epochs 50
+     ```
+   - YOLOv5l Pruned transfer learning:
+     ```bash
+     python train.py --data voc.yaml --cfg ../models/yolov5l.yaml --weights zoo:cv/detection/yolov5-l/pytorch/ultralytics/coco/pruned-aggressive_98 --hyp data/hyp.finetune.yaml --recipe ../recipes/yolov5l.transfer_learn_pruned.md
+     ```
+   - YOLOv5l Pruned Quantized transfer learning:
```
Comment (jeanniefinks) [inline integrations/ultralytics-yolov5/tutorials/yolov5_sparse_transfer_learning.md:133]:
```suggestion
   - YOLOv5l Pruned-Quantized transfer learning:
```

**Review by jeanniefinks — DISMISSED** [review]
Few easy hyphen edits and you're good to go here. Excited to get this out!

**Comment by markurtz** [comment]
> Few easy hyphen edits and you're good to go here. Excited to get this out!

@jeanniefinks updated

**Review by jeanniefinks — APPROVED** [review]

**Review by bfineran — APPROVED** [review]

**Pull request merged**